cirq/google/engine/engine_job.py

```diff
@@ -42,6 +43,9 @@ class EngineJob:
     executing on a machine, or it may have entered a terminal state
     (either succeeding or failing).
 
+    `EngineJob`s can be iterated over, returning `TrialResult`s. These
```

Comment:
Optional: Should we mention that this will force retrieval of the results?  (or is that obvious by returning TrialResults?)

Reply:
done